src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -100,12 +84,41 @@ private ArgumentCaptor(Class<? extends T> clazz) {
      * @return null or default values
      */
     public T capture() {
-        Mockito.argThat(capturingMatcher);
+        return captureIf(Any.ANY);
```

Comment:
Can you write a test (e.g. in [CapturingArgumentsTest](https://github.com/mockito/mockito/blob/master/src/test/java/org/mockitousage/matchers/CapturingArgumentsTest.java)) for the use case you have in mind? The only case I see that will behave differently that now is this:

        ArgumentCaptor<String> argumentCaptor = ArgumentCaptor.forClass(String.class);
        mock.forObject(42);
        mock.forObject("capturedValue");
        verify(mock).forObject(argumentCaptor.capture()); //throws a TooManyActualInvocations-Exeception
     
With the proposed change users won't get a TooManyActualInvocations-Exeception anymore.